### DISCUSSION TITLE
Export metrics externally from Dask Lab extension

### DISCUSSION THREAD

**Discussion by RajeshThallam** [dask/dask#7373] [Q&A] [answered]
Is it possible to export the metrics from [Dask lab extension](https://github.com/dask/dask-labextension) externally to say Cloud Logging/GCS on Google Cloud or CloudWatch/S3 to review utilization after the job is finished?

**Comment by quasiben** [comment, marked as answer]
There is a [metrics endpoint](https://docs.dask.org/en/latest/setup/prometheus.html) which has some information though not everything.  I would also recommend looking at [performance reports](https://distributed.dask.org/en/latest/diagnosing-performance.html#performance-reports)
